src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -151,8 +151,8 @@ void IslandWindow::_HandleCreateWindow(const WPARAM, const LPARAM lParam) noexce
     til::rect rc;
     rc.left = pcs->x;
     rc.top = pcs->y;
-    rc.right = rc.left + pcs->cx;
-    rc.bottom = rc.top + pcs->cy;
+    rc.right = rc.left + pcs->width;
+    rc.bottom = rc.top + pcs->height;

```

Comment:
```suggestion
    rc.right = rc.left + pcs->cx;
    rc.bottom = rc.top + pcs->cy;
```
this seems to be causing your build failure